File: Remonty/wells-back/src/main/java/com/finist/wellsback/models/Vid.java
package com.finist.wellsback.models;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor

@Table(name = "vidy")
public class Vid {

    @Id
    @GeneratedValue
    private Long id;

    @Column(name = "name")
    private String name;

}

File: Remonty/wells-front/src/app/components/vidRemonta/vid/Vid.ts
export class Vid {
    private _id: bigint;
    private _name: string;

    constructor(id: bigint, name: string){
        this._id = id;
        this._name = name;
    }

    get id(): bigint {
      return this._id;
    }

    set id(value: bigint){
      this._id = value;
    }

    get name(): string {
      return this._name;
    }

    set name(value: string){
      this._name = value;
    }
}
